projects/packages/search/src/dashboard/components/record-meter/lib/record-info.js

```diff
@@ -108,8 +108,6 @@ export default function getRecordInfo( postCount, postTypeBreakdown, lastIndexed
  * @returns {object} containing included items with post type and count, and other items, split.
  */
 export function splitUsablePostTypes( postTypeBreakdown, numItems, maxPostTypeCount ) {
-	postTypeBreakdown.sort( ( a, b ) => ( a.data.count[ 0 ] < b.data.count[ 0 ] ? 1 : -1 ) );
```

Comment:
As stated in the description, we sort before passing it to the chart component - where can I find the sorting process please? Seen from here, it's removed❓

Reply:
I guess it's fine even if it's not sorted as long as 'Others' is the last item. Just wondering 😄

Reply:
Is it that the backend already sorts them correctly perhaps? We could check the code to find out.

Reply:
Ah, maybe I could word this better 🤔 

Previously, the post counts were sorted here, as well as using the sort prop in the component. In this PR I have removed both. 

The sorting now only happens as retrieve the post type breakdown from the database in `public_html/wp-content/lib/jetpack-search/class.site-database.php` in lines `105` and `150` by using:

```
		$posts_counts = array_column( $results, 'num_posts', 'post_type' );
		arsort( $posts_counts, SORT_NUMERIC );
```
 which is then passed to the endpoint for use with record meter. So.. I've removed all sorting in the record meter component, so it can fallback to the default API sort provided. How could I better describe this do you think?

Reply:
I would say "remove client-side sorting and rely on API sorting for post types" or similar 🙂